apps/web/pages/availability/[schedule].tsx

```diff
@@ -106,7 +106,7 @@ export default function Availability() {
       isLoading={isPending}
       isSaving={updateMutation.isPending}
       timeFormat={timeFormat}
-      weekStart="Sunday"
+      weekStart={me.data?.weekStart || "Sunday"}
```

Comment:
If there is no user weekStart we assume it to be Sunday and the rest of logic for weekStart is being handled inside of the atom so only thing we need from outside is user week start day